docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 version: '3'
 services:
   senaite:
-      build:
-        context: ./
+      image: mekomsolutions/senaite:latest
```

Comment:
Shouldn't we tag the image?

Reply:
This docker compose file is for testing so we want it to be be the latest.